modules/cnf-about-topology-aware-lifecycle-operator-blocking-crs.adoc

```diff
@@ -0,0 +1,203 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-about-topology-aware-lifecycle-operator-blocking-crs_{context}"]
+= Blocking ClusterGroupUpgrade CRs
+
+You can create multiple `ClusterGroupUpgrade` CRs and control their order of application. 
+
+For example, if you create `ClusterGroupUpgradeC` that blocks the start of `ClusterGroupUpgradeA`, then `ClusterGroupUpgradeA` cannot start until the status of `ClusterGroupUpgradeC` becomes `UpgradeComplete`. 
+
+One `ClusterGroupUpgrade` CR can have multiple blocking CRs.
+
+.Prerequisites
+* Install the Topology Aware Lifecycle Operator.
+* Provision one or more managed clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Create {rh-rhacm} policies in the Hub cluster.
+
+.Procedure
+
+. Create the `ClusterGroupUpgrade` CRs, and then save the YAML files in the `cgu-a.yaml`, `cgu-b.yaml`, and `cgu-c.yaml` files.
++
+[source,yaml]
```

Comment:
I'm thinking it would make more sense to use the examples from here instead: https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/main/tests/kuttl/tests/blocking-mechanisms/01-assert.yaml, to whow the status conditions for blocking CRs, like 
```
status:
  conditions:
  - message: 'The ClusterGroupUpgrade CR is blocked by other CRs that have not yet
      completed: [cgu-c]'
    reason: UpgradeCannotStart
    status: "False"
    type: Ready
```